docs/feature-gates.md

```diff
@@ -538,3 +539,14 @@ Refer to this [document](packetcapture-guide.md) for more information.
 #### Requirements for this Feature
 
 This feature is only supported on Linux for now.
+
+### NFTablesHostNetworkMode
+
+This feature enables Antrea to use nftables instead of iptables to enforce Node host network netfilter rules required by
+functionalities and features that rely on netfilter. Currently, nftables support is limited to the following features:
+
+- AntreaProxy (proxyAll)
+
+#### Requirements for this Feature
+
+This feature is only supported on Linux for now.
```

Comment:
I would remove `for now`. We are never going to implement this on Windows as nftables does not exist on Windows. If we were to introduce an alternative "datapath" on Windows, we would have a dedicated FeatureGate with `Windows` in the name.